forms/tests/conftest.py

```diff
@@ -46,6 +46,11 @@ def django_db_setup(django_db_setup, django_db_blocker):
     fixture_path = Path(__file__).parents[1].parent / "plotsearch/fixtures"
     fixture_filenames = [path for path in fixture_path.glob("*") if not path.is_dir()]
 
+    # Remove field_types.json from fixtures, because it is run in migration.
```

Comment:
Something to double check, I think I configured the tests to run locally without applying the migrations (one by one) to make it faster.
Do the tests apply data migrations in this way when running?

Reply:
Migrations are applied before running the tests. Some tests require running fixtures, but I was able to remove this check as it seems to be unnecessary now.